aws-lambda-java-events/pom.xml

```diff
@@ -50,6 +50,11 @@
             <artifactId>joda-time</artifactId>
             <version>2.10.8</version>
         </dependency>
+        <dependency>
+            <groupId>com.google.code.findbugs</groupId>
+            <artifactId>jsr305</artifactId>
```

Comment:
we shouldn't add that dependency for everyone

Reply:
I've changed the dependency to the jetbrains one. There is no standard for this yet, which is a shame. I think it's important to describe the data of the events as best as we can. I hope that ```@Nullable``` will be an annotation that is added to more events in the future.